Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Builder image
+FROM golang:1.16 as builder
+
+# Workaround because migration tool is private repo
+ARG SSH_PRIVATE_KEY
+RUN mkdir /root/.ssh/ && \
+    echo "${SSH_PRIVATE_KEY}" > /root/.ssh/id_rsa && \
+    chmod 600 /root/.ssh/id_rsa && \
+    ssh-keyscan -t rsa -H github.com > ~/.ssh/known_hosts
+
+ENV GOPRIVATE=github.com/SAP/sap-btp-service-operator-migration
+RUN git config --global url.git@github.com:.insteadOf https://github.com/
+
+# TODO: sent changes to upstream
+# https://github.com/wozniakjan/sap-btp-service-operator-migration/pull/new/allow_in_cluster
```

Comment:
nit: we can get rid of this comment and `TODO` because we are already using the upstream version and not my fork anymore